public/app/features/trails/DataTrailsHome.tsx

```diff
@@ -69,18 +52,28 @@ export class DataTrailsHome extends SceneObjectBase<DataTrailsHomeState> {
           <div className={styles.column}>
             <Text variant="h4">Recent trails</Text>
             <div className={styles.trailList}>
-              {app.state.trail.state.metric && <DataTrailCard trail={app.state.trail} onSelect={model.onSelectTrail} />}
-              {recent.map((trail, index) => (
-                <DataTrailCard key={index} trail={trail.resolve()} onSelect={model.onSelectTrail} />
-              ))}
+              {getTrailStore()
+                .getRecentTrails()
+                .map((trail, index) => {
+                  const resolvedTrail = trail.resolve();
+                  return (
+                    <DataTrailCard
+                      key={resolvedTrail.state.key || index}
+                      trail={trail.resolve()}
```

Comment:
Do we want to use `resolvedTrail` here?

Reply:
not sure that makes sense (for the card). The resolve function is just getter for the underlying object (because scene object cannot exist in multiple places in the scene graph).  For the card this does not matter as it's just a simple react component